packages/admin/blocks-admin/src/blocks/factories/columnsBlock/FinalFormLayoutSelect.tsx

```diff
@@ -112,3 +126,19 @@ const ListItemText = styled(MuiListItemText)`
     margin-top: 0;
     margin-bottom: 0;
 `;
+
+const LayoutDisplayContainer = styled("div")`
```

Comment:
This should also be achievable using the existing components `Menu`, `MenuItem` and `MenuItemContent`. We can have a look at this together